src/QueryM.purs

```diff
@@ -468,14 +471,17 @@ instance Show ClientError where
       <> err
       <> ")"
 
+txToHex :: Transaction -> Effect String
```

Comment:
Maybe we should move this to `Helpers`, since it's also defined in the `Wallet` module

Reply:
Actually that might lead to cyclical module imports unfortunately